dev/update-alloc-limits-to-last-completed-ci-build

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set -eu
+set -o pipefail
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+url_prefix=${1-"https://ci.swiftserver.group/job/swift-nio-http2-2-"}
+target_repo=${2-"$here/.."}
+tmpdir=$(mktemp -d /tmp/.last-build_XXXXXX)
+
+for f in swift51 swift52 swift53 swift54 swift55 nightly; do
```

Comment:
With https://github.com/apple/swift-nio/pull/1860 in mind, should we drop `swift51` from this list?

Reply:
For now I'd like to keep it: when we make the equivalent PR here, we'll need to remove 51 from this script, just as the linked PR does.

Reply:
Should we add `swift50` then since we still track allocations for it in http/2?

Reply:
Nope, we'll just stop tracking allocs for it.

Reply:
Even better!